_translator-files/po/nb_NO/Getting-Started.po

```diff
@@ -141,18 +141,22 @@ msgstr ""
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:53
-#, no-wrap
+#, fuzzy, no-wrap
 msgid ""
 "<figure>\n"
 "  <img src=\"{% include img/en-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
 "  <figcaption>Let people know who you are</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"  <img src=\"{% include img/nb-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"Skjermavbildning\" style=\"border: 5px solid grey;\">\n"
```

Comment:
"Screenshot" ("Skjermavbildning") shouldn't be translated - just the 'figcaption'.

Reply:
Is there a reason why? SEO and assumption people with screen-readers want Norwegian Bokmål (as opposed to English) when selecting the nb_NO locale is why I translated it.

Reply:
I may have been mistaken in assuming it shouldn't be translated.